### ISSUE TITLE
Rename OSPSuite.InstallationValidator to InstallationValidator

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/InstallationValidator#68]
OSPSuite. is reserved for components but it does not feel right for a tool

**Comment by msevestre** [comment]
@rwmcintosh @Yuri05 thoughts?

**Comment by rwmcintosh** [comment]
Although it does validate the installation of OSPSuite.

**Comment by msevestre** [comment]
Yeah but its installed as part of the suite install. We do not call
OSPSuite.PK-Sim or OSPSuite.MoBi

**Comment by rwmcintosh** [comment]
It was an attempt at humour that really never stood a chance.

I meant that in plain English you might say 'osp suite installation
validator' just like the contraction. That's different for the other tools.

But it is inconsistent anyway.

**Comment by msevestre** [comment]
Maybe if you say it out loud?:)

**Comment by Yuri05** [comment]
Let's rename

**Comment by msevestre** [comment]
Sounds good
